### PR TITLE
NE-679: Release note addition 

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.10 only
- [Jira](https://issues.redhat.com/browse/NE-807)
- [Preview](https://deploy-preview-42713--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes)
- @lihongan  ptal and verify

### CODE REVIEW

**Comment by lihongan** [comment]
/lgtm
Thank you @Amrita42

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -530,6 +530,12 @@ You can now set the maximum length of the syslog message in the Ingress Controll
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress Controller configuration parameters].
 
+[id="ocp-4.10-ne-set-coredns-forwarding-policy"]
+==== Set CoreDNS forwarding policy
+You can now set the CoreDNS forwarding policy through the DNS operator. The default value is `Random`, you can also set the values to `RoundRobin`, and `Sequential`.
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:535]: I don't think you need a comma between `RoundRobin` and `Sequential`.

**Review by opayne1 — COMMENTED** [review]
Just one small nit. Otherwise, lgtm!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (Amrita42) [reply release_notes/ocp-4-10-release-notes.adoc:535]: removing the comma

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -530,6 +530,12 @@ You can now set the maximum length of the syslog message in the Ingress Controll
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress Controller configuration parameters].
 
+[id="ocp-4.10-ne-set-coredns-forwarding-policy"]
+==== Set CoreDNS forwarding policy
+You can now set the CoreDNS forwarding policy through the DNS operator. The default value is `Random`, you can also set the values to `RoundRobin` and `Sequential`.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-10-release-notes.adoc:535]:
```suggestion
You can now set the CoreDNS forwarding policy through the DNS Operator. The default value is `Random`, and you can also set the value to `RoundRobin` or `Sequential`. 
```

**Pull request merged**